velox/docs/monitoring/stats.rst

```diff
@@ -18,24 +18,50 @@ analysis.
 
 Memory Arbitration
 ------------------
+These stats are reported by all operators.
+
 .. list-table::
    :widths: 40 10 50
    :header-rows: 1
 
    * - Stats Name
-     - Type
+     - Unit
      - Description
    * - memoryReclaimCount
-     - kNone
+     - <empty>
```

Comment:
Not sure we name this as kCount type?

Reply:
Can you use an actual empty string here?